code/Reimplementation/FeatureExtractor/LPCExtractor.py

```diff
@@ -1,12 +1,15 @@
 from FeatureExtractor.ExtractorInterface import ExtractorInterface
-
+from multiprocessing import Pool
 import librosa
 
+def calculate_lpc(frame, sr, order):
```

Comment:
Warum ist die Funktion nicht mehr Teil der Klasse?
Verstehe warum du es als extra funktion machst, aber dann packs doch in die Klasse, dann ist alles beisammen....